packages/block-editor/README.md

```diff
@@ -302,10 +302,6 @@ _Related_
 
 -   <https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/inspector-controls/README.md>
 
-<a name="LinkControl" href="#LinkControl">#</a> **LinkControl**
-
-Undocumented declaration.
-
```

Comment:
noting this was removed automattically by a `docs:build` script because all components containing word `unstable` or `experimental` are filtered there

Reply:
Ok